src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -92,12 +92,14 @@
                          :wakuV2Nameserver         "1.1.1.1"
                          :logLevel                 (when log-enabled? config/log-level)
                          :logEnabled               log-enabled?
-                         :logFilePath              (status/log-file-path)
+                         :logFilePath              (status/log-file-directory)
```

Comment:
i think we should reference [updateConfig](https://github.com/status-im/status-react/blob/006b11d508f76bf0ef23fdb18af5f1ab39f98309/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java#L216) , and we need add extra param `logDir`, there is a such logical in [status-go](https://github.com/status-im/status-go/blob/f6e01724fd102fa0b20da1daa9f2f5337b1df862/api/geth_backend.go#L1057) , have `logFilePath` is not enough

Reply:
mmh, not sure I follow, before we sent 2 parameters, the filename & the path, now we only send a single one, since I don't think there's need to send the filename, it's fine to hardcode to `geth.log`, I checked on android and the two nodeconfig are identical, but I think I am misunderstanding your point :) , would you mind elaborating please?

Reply:
according to [this](https://github.com/status-im/status-go/blob/e9482e3974a2f82bd6a23f0c9e861b5bca23f472/api/geth_backend.go#L1057) , if we only set nodeConfig. LogFile and nodeConfig.LogDir is empty, nodeConfig.LogFile will be overwrite with `rootDataDir+nodeConfig.LogFile , that's not we want right? rootDataDir is already a absolute path, nodeConfig.LogFile you passed(status/log-file-directory) is also a absolute path, so you know .. :) @cammellos